dashboard/app/views/courses/index.html.haml

```diff
@@ -23,5 +23,9 @@
   = render file: Rails.root.join('..', 'shared', 'haml', 'course_explorer_table.haml')
   = render file: Rails.root.join('..', 'shared', 'haml', 'tool_explorer_table.haml')
   = render partial: 'home/all_courses'
+  -if current_user
+    - unless current_user.teacher?
+      #user_hero
+        = render template: 'api/user_hero'
```

Comment:
That said, if we do have to leave in the ruby code, we could make it one line with `if current_user && !current_user.teacher?`